portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,43 +20,68 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import java.util.ArrayList;
+import java.util.*;
```

Comment:
nit: alphabetize the list of import statements

Reply:
fixed!